roles/zammad/tasks/install.yml

```diff
@@ -30,18 +30,28 @@
   when: "ansible_distribution | lower == 'ubuntu'"
   block:
 
-    - name: "Install | Add Zammad apt key"
+    - name: "Remove Zammad apt key from legacy trusted.gpg keyring"
       ansible.builtin.apt_key:
         url: "https://dl.packager.io/srv/zammad/zammad/key"
-        state: "present"
+        state: "absent"
 
     - name: "Install | Add Zammad DEB repository"
-      ansible.builtin.apt_repository:
```

Comment:
Like in GitLab. Should we remove this as well?

Reply:
Of course, I missed that.